docs/conf.py

```diff
@@ -62,3 +62,22 @@
 # sphinx/builders/linkcheck.py:86: RemovedInSphinx80Warning: The default value for 'linkcheck_report_timeouts_as_broken' will change to False in Sphinx 8, meaning that request timeouts will be reported with a new 'timeout' status, instead of as 'broken'. This is intended to provide more detail as to the failure mode. See https://github.com/sphinx-doc/sphinx/issues/11868 for details.
 #   warnings.warn(deprecation_msg, RemovedInSphinx80Warning, stacklevel=1)
 linkcheck_report_timeouts_as_broken = False
+
+# Enable LaTeX PDF generation
+extensions += ["sphinx.ext.imgmath"]
```

Comment:
I am seeing the following warning in the docs build. RtD will treat warnings as errors.

> WARNING: LaTeX command 'latex' cannot be run (needed for math display), check the imgmath_latex setting